api-server/llama-api-server/src/main.rs

```diff
@@ -68,6 +68,9 @@ struct Cli {
     /// Number of layers to run on the GPU
     #[arg(short = 'g', long, default_value = "100")]
     n_gpu_layers: u64,
+    /// Disable memory mapping for file access
+    #[arg(long)]
```

Comment:
The option is only available for chat models, right? If so, it would be better to add this info to the comment. For example, 

```text
/// Disable memory mapping for file access. Only available for chat models.
```

Reply:
Can I change the description to "Disable memory mapping for file access of chat models" ? By first glance, I think this would be more brief and descriptive than "Only available for chat model.".

Reply:
LGFM. Thanks!